### DISCUSSION TITLE
DB Queries fail silently (from SvelteKit example)

### DISCUSSION THREAD

**Discussion by r25s** [supabase/supabase#14795] [Questions]
Hi all!

I was doing the SvelteKit example tutorial (https://supabase.com/docs/guides/getting-started/tutorials/with-sveltekit) and I got stuck at the final step (Account Page) because my form was neither updating nor reading from the database. But it was also not producing any errors!

I checked that the `update` action in `accounts/+page.server.ts` was being called, which it was. Upon further investigation I realized that my database schema lacked the `full_name` field - and that's because I set up the `profiles` table many months ago, (perhaps before the tutorial existed). So when I had run the "User Management Starter" SQL script at the time, no `full_name` field was created.

Realizing this I commented out the every reference to `full_name` in `+page.server.ts`, (see it here for reference: https://supabase.com/docs/guides/getting-started/tutorials/with-sveltekit#account-page) it was only then that the queries in the `update` action and the `load()` function were completed successfully.

My problem is that the queries failed silently. I got no warning in the console that there was an error. Plus the `update` action checks if an error is return from the query at which point it would `return fail(500)`. But I got no error at all from either the `update` action or `load()` function.

I posted here because I'm not sure if this should be a bug report, or if I'm just missing something.

Thanks!

**Comment by silentworks** [comment]
This isn't a bug as it's just how this particular tutorial was developed. In your application you would factor in all the errors and figure out how best to handle them. The tutorial was about using the feature and no so much about error handling.

**Reply by r25s** [reply]
Thanks for your reply. The issue is not about the lack of error _handling_ in the app itself, it's about the lack of an _error_ from the `upsert` function. Instead it failed silently, which is highly undesirable. It would be better if the function threw an error an crashed the app.
